File: src/tool/spring-boot-generator/output/contractSign/contractSign/src/main/java/com/contractSign/pojo/entity/ContractMain.java
package com.contractSign.pojo.entity;

@packages
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;
@packages


@ApiModel("")
@Getter
@Setter
public class ContractMain @extends {

    @ApiModelProperty("合同编号")
    private Integer id;

    @ApiModelProperty("合同发起人")
    private Integer userId;

    @ApiModelProperty("合同类型")
    private Integer contractType;

    @ApiModelProperty("合同标题")
    private String contractTitle;

    @ApiModelProperty("原始合同文件")
    private String rawContractFile;

    @ApiModelProperty("原始合同Hash")
    private String rawContractHash;

    @ApiModelProperty("签约截止日期")
    private Date signExpireTime;

    @ApiModelProperty("合同状态")
    private Integer contractState;

    @ApiModelProperty("签约合同文件")
    private String signContractFile;

    @ApiModelProperty("签约合同Hash")
    private String signContractHash;

    @ApiModelProperty("合同生效时间")
    private Date effectTime;

    @ApiModelProperty("合同失效时间")
    private Date expireTime;

    @ApiModelProperty("合同有效期")
    private Integer validPeriod;

    @ApiModelProperty("合同价格")
    private Integer contractPrice;

    @ApiModelProperty("创建时间")
    private Date createTime;

}

File: src/tool/spring-boot-generator/output/contract/db/contractMain.js
let contractMain = {
	name: "contractMain",
	description: "",
	columns: {
		contractPrice: {
			name: "contractPrice",
			description: "合同价格",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		contractState: {
			name: "contractState",
			description: "合同状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		contractTitle: {
			name: "contractTitle",
			description: "合同标题",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		contractType: {
			name: "contractType",
			description: "合同类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		effectTime: {
			name: "effectTime",
			description: "合同生效时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		expireTime: {
			name: "expireTime",
			description: "合同失效时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		id: {
			name: "id",
			description: "合同编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		rawContractFile: {
			name: "rawContractFile",
			description: "原始合同文件",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		rawContractHash: {
			name: "rawContractHash",
			description: "原始合同Hash",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signContractFile: {
			name: "signContractFile",
			description: "签约合同文件",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signContractHash: {
			name: "signContractHash",
			description: "签约合同Hash",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signExpireTime: {
			name: "signExpireTime",
			description: "签约截止日期",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		userId: {
			name: "userId",
			description: "合同发起人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		validPeriod: {
			name: "validPeriod",
			description: "合同有效期",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"contractPrice",
	"contractState",
	"contractTitle",
	"contractType",
	"createTime",
	"effectTime",
	"expireTime",
	"id",
	"rawContractFile",
	"rawContractHash",
	"signContractFile",
	"signContractHash",
	"signExpireTime",
	"userId",
	"validPeriod"
];

function create() {
  return {};
}

contractMain.columnsArray=columnsArray;
contractMain.create=create;
module.exports=contractMain;